Doc/reference/compound_stmts.rst

```diff
@@ -245,13 +245,17 @@ handler is started. This search inspects the :keyword:`!except` clauses in turn
 until one is found that matches the exception.
 An expression-less :keyword:`!except` clause, if present, must be last;
 it matches any exception.
-For an :keyword:`!except` clause with an expression,
-that expression is evaluated, and the clause matches the exception
-if the resulting object is "compatible" with the exception.  An object is
-compatible with an exception if the object is the class or a
-:term:`non-virtual base class <abstract base class>` of the exception object,
-or a tuple containing an item that is the class or a non-virtual base class
-of the exception object.
+
+For an :keyword:`!except` clause with an expression, the
+expression must evaluate to a type or a tuple containing types. Each type must
+be a subclass of :exc:`BaseException`; otherwise, the runtime will raise a
```

Comment:
It can be BaseException itself. Can this be worded to make that clear?

Reply:
clarified to "must be :exc:`BaseException` or a subclass of :exc:`BaseException`"

same for the BaseExceptionGroup mention in the except* section